js/components/contextMenu.js

```diff
@@ -157,6 +157,11 @@ class ContextMenuItem extends ImmutableComponent {
             style={iconStyle}></span>
           : null
       }
+      {
+        this.hasSubmenu
+        ? <span className='submenuIndicator contextMenuIcon bookmarkIcon fa fa-folder-o' style={iconStyle} />
```

Comment:
I suggest to modify `js/contextMenu.js` and look in `bookmarkItemsInit`. That's where icons are set. If you find it is a folder you can add a new property like faIcon and then if it is specified inside `js/compnents/contextMenu.js` you can use it.